test/tests/nodalkernels/constraint_enforcement/interpolated-ncp-lm-nodal-enforcement-nodal-forces.i

```diff
@@ -0,0 +1,122 @@
+l=10
+nx=100
+num_steps=10
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  xmax = ${l}
+  nx = ${nx}
+[]
+
+[Variables]
+  [u]
+  []
+  [lm]
+  []
+[]
+
+[ICs]
+  [u]
+    type = FunctionIC
+    variable = u
+    function = '${l} - x'
+  []
+[]
+
+[Kernels]
+  [time]
+    type = TimeDerivative
+    variable = u
+  []
+  [diff]
+    type = Diffusion
+    variable = u
+  []
+  [ffn]
+    type = BodyForce
+    variable = u
+    function = '-1'
+  []
+[]
+
+[NodalKernels]
+  [positive_constraint]
+    type = LowerBoundNodalKernel
+    variable = lm
+    v = u
+    exclude_boundaries = 'left right'
+  []
+  [forces]
+    type = CoupledForceNodalKernel
+    variable = u
+    v = lm
+  []
+[]
+
+
+[BCs]
+  [left]
+    type = DirichletBC
+    boundary = left
+    value = ${l}
+    variable = u
+  []
+  [right]
+    type = DirichletBC
+    boundary = right
+    value = 0
+    variable = u
+  []
+[]
+
+[NodalKernels]
+[]
```

Comment:
Do you need this empty `NodalKernels` block? You already have one `NodalKernels` earlier.